README.md

```diff
@@ -1,61 +1,92 @@
-	};
-	function findImports(path) {
-	   if (path === 'lib.sol')
-	      return { contents: 'library L { function f() returns (uint) { return 7; } }' }
-	   else
-	      return { error: 'File not found' }
-	}
-	var output = solc.compile({sources: input}, 1, findImports);
-	for (var contractName in output.contracts)
-		console.log(contractName + ': ' + output.contracts[contractName].bytecode);
+```javascript
+var solc = require('solc');
+var input = {
+	'cont.sol': 'import "lib.sol"; contract x { function g() { L.f(); } }'
+};
+function findImports(path) {
+	if (path === 'lib.sol')
+		return { contents: 'library L { function f() returns (uint) { return 7; } }' }
+	else
+		return { error: 'File not found' }
+}
+var output = solc.compile({sources: input}, 1, findImports);
+for (var contractName in output.contracts)
+	console.log(contractName + ': ' + output.contracts[contractName].bytecode);
+```
+
+
+### Using a Legacy Version
+
+In order to allow compiling contracts using a specific version of Solidity, the `solc.useVersion` method is available. This returns a new solc object using the version provided. **Note**: version strings must match the version substring of the files availble in `/bin/soljson-*.js`. See below for an example.
 
+```javascript
+var solc = require('solc');
+// by default the latest version is used
+// ie: solc.useVersion('latest')
 
-###Using a legacy version
+// getting a legacy version
+var solcV011 = solc.useVersion('v0.1.1-2015-08-04-6ff4cd6');
+var output = solcV011.compile('contract t { function g() {} }', 1);
+```
 
-In order to allow compiling contracts using a specific version of solidity, the `solc.useVersion` method is available. This returns a new solc object using the version provided. **Note**: version strings must match the version substring of the files availble in `/bin/soljson-*.js`. See below for an example.
+If the version is not available locally, you can use `solc.loadRemoteVersion(version, callback)` to load it directly from GitHub.
 
-	var solc = require('solc');
-	// by default the latest version is used
-	// ie: solc.useVersion('latest')
+You can also load the "binary" manually and use `setupMethods` to create the familiar wrapper functions described above:
+`var sol = solc.setupMethods(require("/my/local/soljson.js"))`.
 
-	// getting a legacy version
-	var solcV011 = solc.useVersion( 'v0.1.1-2015-08-04-6ff4cd6' );
-	var output = solcV011.compile( "contract t { function g() {} }", 1 );
+### Using the Latest Development Snapshot
+
+By default, the npm version is only created for releases. This prevents people from deploying contracts with non-release versions because they are less stable and harder to verify. If you would like to use the latest development snapshot (at your own risk!), you may use the following example code.
+
+```javascript
+var solc = require('solc');
+
+// getting the development snapshot
+var solcSnapshot = solc.loadRemoteVersion('latest', function(err, solcSnapshot) {
```

Comment:
`var solcSnapshot = ` is not needed, please remove (sorry for having it my example)